kong/api/routes/kong.lua

```diff
@@ -90,15 +90,17 @@ return {
           connections_handled = tonumber(handled),
           total_requests = tonumber(total)
         },
-        database = {}
+        database = {
+          reachable = false
+        }
       }
 
-      for k, v in pairs(dao.daos) do
-        local count, err = v:count()
-        if err then
-          return helpers.responses.send_HTTP_INTERNAL_SERVER_ERROR(err)
-        end
-        status_response.database[k] = count
+      -- ping DB
+      local err
+      status_response.database.reachable, err = dao.db:is_reachable()
```

Comment:
Due to the below changes with the return value (and because this `reachable()` function might be used elsewhere and should not be tied to this endpoint for the sake of modularisation), this could become:
```
local ok, err = dao.db:reachable()
if not ok then
  ngx.log(ngx.ERR, "msg...", err)

else
  status_response.database.reachable = true
end
```